gui/cache.go

```diff
@@ -48,8 +48,8 @@ type Cache struct {
 	blockExplorerURL string
 	minedWork        []minedWork
 	minedWorkMtx     sync.RWMutex
-	workQuotas       []workQuota
-	workQuotasMtx    sync.RWMutex
+	dividends        []dividend
```

Comment:
Hmm not sure about this change considering the values being reported here are not actual amounts being paid to participating accounts but percentages of the work contributed so far towards finding the next block. Maybe `work quota` is also not descriptive or simple enough, but using `dividend` here is misleading.

Reply:
Renamed to RewardQuota as discussed on Matrix